backend/matchmaker_service.py

```diff
@@ -0,0 +1,65 @@
+import firebase_admin
+from firebase_admin import credentials, firestore
+from game import Game
+from game_engine.common import GameOptions
+from game_engine.firestore import FirestoreDB
+import time
+import threading
+
+_FIRESTORE_PROD_CONF_JSON_PATH = ''
+_TEST_FIRESTORE_INSTANCE_JSON_PATH = '../firebase/stv-game-db-test-4c0ec2310b2e.json'
+json_config_path = _TEST_FIRESTORE_INSTANCE_JSON_PATH
+
+
+
+class MatchmakerService:
+    def __init__(self, json_config_path=json_config_path, region="US", min_players=5, is_test=True):
+        self._gamedb = FirestoreDB(json_config_path=json_config_path)
+        self._min_players = min_players
+        self._region=region
+        self._is_test = is_test
+        self._STOP = False

```

Comment:
nit: Update 'self._STOP' to 'self._stop' and set its type to threading.Event() instead of boolean.